src/shell/scripts/omp.bash

```diff
@@ -54,6 +54,11 @@ function _omp_hook() {
     local omp_stack_count=$((${#DIRSTACK[@]} - 1))
     local omp_elapsed=-1
     local no_exit_code="true"
+    local omp_columns=0
+
+    if [[ -n "$COLUMNS" ]]; then
```

Comment:
Just a note that `-n` is the default test, so there is no need to specify it. That is, `[[ -n $COLUMNS ]]` is the same as `[[ $COLUMNS ]]`. Moreover, since `[[` is a language construct (as opposed to `[`), it is not vulnerable to word splitting, so there is no need to quote if you don't want to (though there is no harm in doing so).

Reply:
I learned something today! Thanks 🙏🏻